extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-serviceaccount.yaml

```diff
@@ -0,0 +1,50 @@
+{{ if .Values.webhook.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.webhook.serviceAccount.name }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-delete-policy": hook-failed, before-hook-creation
+    "helm.sh/hook-weight": "-10"
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: config-store-webhook-clusterrole
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
+    "helm.sh/hook-delete-policy": hook-failed, before-hook-creation
+    "helm.sh/hook-weight": "-10"
+rules:
+- apiGroups: ["batch"]
+  resources: ["jobs"]
+  verbs: ["delete", "deletecollection"]
+- apiGroups: [""]
+  resources: ["secrets"]
```

Comment:
Q/S: Don't we only need secrets access if we're generating the certs?

Reply:
Updated to make secret permissions conditional.